### DISCUSSION TITLE
Arbitrary groups not working on my side but does in Taillwind Play

### DISCUSSION THREAD

**Discussion by marcdegagne** [tailwindlabs/tailwindcss#10328] [Help]
Hello,

I am experiencing with group and peer from this page https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state.

I find it as a good idea instead of having to create a @layer components {}. That is how I found myself solving some issues on parent state child interaction.

While it works fine when using the speudo-classes like group-hover and others (not tested yet), I found that arbitrary groups do not work on my side out of the box but do on Tailwind Play. See example: https://play.tailwindcss.com/QNrD7IVMGP

Found some potential solutions to make it work but require some custom JS as these two solutions:

- https://play.tailwindcss.com/JSu4IS72pj?file=config
- https://stackoverflow.com/questions/73884269/custom-group-states-in-tailwind-css

**Why it does work out of the box on Tailwind Play but not on my side?**
I do have the same version installed as the one tested with, which is the latest.

Thanks

**Comment by adamwathan** [comment]
Hey! Can you create a repository where it doesn't work?

**Reply by marcdegagne** [reply]
Hello Adam,

I will have to try first with a fresh install of Laravel and basic requirements of this project to reproduce it as much as I can without exposing sensitive code. This is a project to be delivered.
Note that I am not yet very familiar with Github or Git, this may take me a while... :-(

**Reply by marcdegagne** [reply]
I wanted to rule first that this is not a system specific so I have created two new projects. 

One with Laravel 8.83 that still use mix and installed Tailwindcss.
Another with Laravel 9.x (latest) that use Vite and installed Tailwindcss.

Both are working and compile "group" as expected.

On the current  project, I ruled out also all other scripts that could interfere by testing a blank page just using "css/app.css" and the group syntax to test. Not working.

I verified the tailwind.config, package.json, webpack.mix, postcss.config and updated some package to same as the laravel 8.83.

So now I know this is package specific. I will prep a copy to a repo with a test page.

**Comment by marcdegagne** [comment]
The problem was 36 inches from the screen!
After updading to the latest possible working packages and runing npm, I forgot to stop and restart npm run watch.

That did the trick! It's now working.

That being said, in the process I updated tailwind to the latest version 3.2.4. Thought I had the latest. 

Thanks
